src/commands/serve.ts

```diff
@@ -70,7 +70,8 @@ export class ServeCommand implements Command {
       browser: options['browser'],
       openPath: options['open-path'],
       npm: options['npm'],
```

Comment:
this should pull from config as well, right?

Reply:
yeah; will update